src/main/java/com/jcabi/github/RtReleases.java

```diff
@@ -113,6 +112,30 @@ public Release get(final int number) {
     }
 
     @Override
+    public Release tagged(final String name) throws IOException {
+        return this.get(
+                this.request.uri().path("/tags/").path(name)
```

Comment:
@weisebrazil we generally follow a rule that each and every indentation is exactly 4 spaces more than the previous line, here you added 8 spaces, same thing for the lines below